.github/workflows/perf.yml

```diff
@@ -147,28 +148,26 @@ jobs:
         with:
           path: artifacts
           
-      - name: Produce performance model table
-        run: python e2etest/get-performance-model-table.py
-      
-      - name: Create a new branch
-        uses: peterjgrainger/action-create-branch@v2.0.1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          branch: "perf-test-${{ github.run_id }}"
-          
-      - name: Commit to a branch
-        uses: stefanzweifel/git-auto-commit-action@v4
-        with:
-          commit_message: "Update benchmarking"
-          branch: "perf-test-${{ github.run_id }}"
-          file_pattern: docs/performance_model.md
-      
-      - name: pull-request
-        uses: repo-sync/pull-request@v2
+      - name: Produce performance report
+        run: python e2etest/get-performance-model-table.py ${{ needs.get-testing-version.outputs.testing_version }}
+
+      - name: Store benchmark result
+        uses: jefchien/github-action-benchmark@v1.11.12-alpha2
```

Comment:
Anyway for us to know the parameters that are required in your private GitHub action thought?

Reply:
It's a fork of the existing https://github.com/benchmark-action/github-action-benchmark. The change was needed to pass the workflow dispatch input SHA into the action so the benchmarks are tagged with the correct commit. I have a PR https://github.com/benchmark-action/github-action-benchmark/pull/93 open for the change, but until that gets merged or another method of inserting the commit is found, we'll need to use the fork.

Reply:
Got cha. Make more sense to me now. Currently reviewing yours other PR as well.

Reply:
Please add a comment